src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -374,6 +374,13 @@
                     "pattern": "The submission limit must be and integer between 1 and 500!",
                     "required": "The submission limit must be set!"
                 }
+            },
+            "diffReport": {
+                "button": "Show difference between the template and solution repositories",
+                "tooltip": "Shows the git-diff between the template and solution repositories.™",
```

Comment:
Is hte ``™``intended? :D

Reply:
Nope. Looks like I accidentally hit the key combination for it.